pkg/agent/baker.go

```diff
@@ -1264,6 +1264,31 @@ root = "{{GetDataDir}}"{{- end}}
     {{- if IsKata }}
     [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata]
       runtime_type = "io.containerd.kata.v2"
+    [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata]
```

Comment:
is this something that should be added separately?

Reply:
yep, is this valid for regular kata images too?

Reply:
Yes, the same change would apply to regular kata release image as well.
You are right. This change has been moved to #3378 where it would be more fitting.